docs/manifest.json

```diff
@@ -1103,22 +1103,16 @@
 		"markdown_source": "../packages/components/src/navigator/navigator-button/README.md",
 		"parent": "components"
 	},
-	{
-		"title": "NavigatorProvider",
-		"slug": "navigator-provider",
-		"markdown_source": "../packages/components/src/navigator/navigator-provider/README.md",
-		"parent": "components"
-	},
```

Comment:
FYI, removing things from the manifest is not going to remove it from the [Block Editor Handbook](https://developer.wordpress.org/block-editor/reference-guides/components/) 😱 There are some extra steps, see https://github.com/WordPress/gutenberg/pull/60003#issuecomment-2010288927

Reply:
Thank you! I'll follow-up accordingly

Reply:
Hey @WordPress/wordpress-org-meta-team , could you help with this? We need to update the Block Editor Handbook, removing the following components:

- [`NavigatorProvider`](https://developer.wordpress.org/block-editor/reference-guides/components/navigator-provider/)
- [`NavigatorButton`](https://developer.wordpress.org/block-editor/reference-guides/components/navigator-button/)
- [`NavigatorBackButton`](https://developer.wordpress.org/block-editor/reference-guides/components/navigator-back-button/)
- [`NavigatorScreen`](https://developer.wordpress.org/block-editor/reference-guides/components/navigator-screen/)
- [`NavigatorToParentButton`](https://developer.wordpress.org/block-editor/reference-guides/components/navigator-to-parent-button/)


If possible, we'd also like to set a redirect for those pages being removed. The redirect should point to [the `Navigator` component](https://developer.wordpress.org/block-editor/reference-guides/components/navigator/).


Could you please help with that? Thank you 🙏

Reply:
These updates have shipped. I noticed the formatting on the new doc loses consistency around the `Navigator.Button` point (heading is not bold):

![Screenshot 2024-10-01 at 10 42 06 AM](https://github.com/user-attachments/assets/37da3574-ee2c-4830-9ed8-fe50b6d62564)

Reply:
Thank you!



> I noticed the formatting on the new doc loses consistency around the `Navigator.Button` point (heading is not bold):

Good catch! Opened https://github.com/WordPress/gutenberg/pull/65763 to fix it